package.json

```diff
@@ -1291,7 +1291,7 @@
                     "default": [],
                     "examples": [
                         [
-                            "**/*.js",
+                            "*.js",
```

Comment:
I would keep the `**/*.js` and just add another like `*.cache` or something.

Reply:
That's good, too. Shall I submit it?

Reply:
Yes Please